### PR TITLE
New: Input package. Cleanup: main.py, cart.py

### PR DESCRIPTION
**Pull request by amrit-choudhary**
Added a new input package, which is easier to use than using the default pygame input system.
You can use WASD or arrow keys to move.

### CODE REVIEW

**Comment by amrit-choudhary** [comment]
You can now use WASD or arrow keys to move

**Comment by amrit-choudhary** [comment]
made all of the project pep8 compliant.
Only 'line too long' errors remain at some place.  Which a lot of developers ignore.
https://code.djangoproject.com/ticket/23395

File: src/input_package/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Input Package User Guide
+This package provides a wrapper around the basic pygame input functionality to enable easy to use input system for games.
+Main features
+- Easy to use
+- Multiple keys for same button action (ex. ctrl or F to shoot)
```
Comment (vineetjc) [inline src/input_package/README.md]: Multiple keys for same button action (ex. ctrl or F to shoot)?? (I'm assuming this is from somwhere else :p )

**Comment by vineetjc** [comment]
Majority of the changes are awesome.

I think in some places, scroll mouse is still happening, so I guess I'll just expect that in the next PR

**Pull request merged**